src/partials/about.html

```diff
@@ -44,6 +43,11 @@ <h3 class="about-list-header">04</h3>
       </ul>
       <div class="about-picture">
         <picture>
+          <source srcset="
+              ../img/memoji.png 1x,
+              ../img/memoji@2x.png 2x
+            " />
+          <img class="about-picture-img" src="../img/memoji.png" alt="Smiling girl pointing her finger" />
```

Comment:
wrong path, check where the index file is located relative to the images, so "./", not "../"